assignments/introductory-assignments.md

```diff
@@ -169,6 +245,24 @@ Reference: [ASCII Table](ascii-table.md)
 - What is the decimal representation of your first name?
 - What is the ASCII representation of the decimal numbers 115, 101, 108, 102, 105, 101?
 
+## Unicode
+
+So the 8-bit sequence for a character is translated through ASCII. Still, only 128 different possible states can we get with the ASCII table because ASCII is represented through 8-bits, but the leading bit is always 0. That means we can only represent 128 different characters. So the ASCII encoding wastes one leading bit. That is why we prefer to deal with full bytes (8 bits) on a computer.
```

Comment:
```suggestion
So the 8-bit sequence for a character is translated through ASCII. Still, only 128 different possible states can we get with the ASCII table because ASCII is represented through 8-bits, but the leading bit is always 0. That means we can only represent 128 different characters. So the ASCII encoding wastes one leading bit. That is why we prefer to deal with full bytes (8 bits) on a computer.
```

7 bit, huete werden 8 bit verwendent, deswegen gibt es utf8, backwards compatible with ascii, most significant bit weird 0 gesetzte, deshalb backwards compatibility. Ascii is 7 bit, computer byte wise 8 bit, deswegen 8 bit basiertes encoding utf8, backward compatible mit ascii, most significant bit is 0.

The 8-bit sequ represe ... to the ascii standard. ASCII only maps 7 bits to character 
through eight bits (hyven weg?)

Reply:
yes

Reply:
I rewrote the description, what do you think?